### PR TITLE
Badge component added

### PR DESCRIPTION
**Pull request by nareshpoonia**
Badge component added

### CODE REVIEW

File: html/badge.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (MaruthiPothuganti) [inline html/badge.html]: Add a Title

File: styles/badge.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import url(/styles/docs.css);
+@import url(/styles/index.css);
+
+.icon-button {
+  position: relative;
+  width: 5rem;
```
Comment (abhij1607) [inline styles/badge.css:6]: You can define and use variables instead of absolute values

File: styles/docs.css [hunk 3]
```diff
@@ -33,6 +34,12 @@ code {
 .mt-3 {
   margin-top: 3rem;
 }
+.flex-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```
Comment (MaruthiPothuganti) [inline styles/docs.css:42]: These can be moved into utility css as these are used regularly

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes. Everything else looks good.

File: styles/badge.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  height: 5rem;
+  color: var(--icon-color);
+  background: var(--icon-background-color);
+  border: none;
+  outline: none;
```
Comment (abhij1607) [inline styles/badge.css]: I think, outline: none is not considered to be good practice..might look for an alternative, I can't remember at this moment

**Review by abhij1607 — APPROVED** [review]
Except the comments mentioned, everything seems nice
Very clean code, can be merged

File: styles/badge.css (hunk 2 above)
Comment (nareshpoonia) [reply styles/badge.css:6]: It is a big value which probably wouldn't be used in the future, for all the common values, I'll be using variables instead.
Thanks for the suggestion

File: styles/docs.css (hunk 3 above)
Comment (nareshpoonia) [reply styles/docs.css:42]: I don't have a separate utility classes CSS file, will make it soon & shift them there.

File: html/badge.html (hunk 1 above)
Comment (nareshpoonia) [reply html/badge.html]: Added Title

File: styles/badge.css (hunk 4 above)
Comment (nareshpoonia) [reply styles/badge.css]: Use 1px, transparent instead

**Pull request merged**